### PR TITLE
fix problem labels

### LINKED ISSUE #4625: Contact Form Submission - Mistake (Silver - Functional Graphs - IOI Tropical Gardens)
Someone submitted the contact form!

**URL**: https://usaco.guide/dashboard/
**Module**: Silver - Functional Graphs - IOI Tropical Gardens
**Topic**: Mistake
**Message**: 
https://usaco.guide/silver/func-graphs#problem-IOI11_garden

has internal solution when you click on the triple dots, but once clicked nothing shows up

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

close #4625
we're so cooked

### CODE REVIEW

File: content/6_Advanced/SCC.problems.json [hunk 1]
```diff
@@ -153,7 +153,7 @@
       "isStarred": false,
       "tags": ["2SAT"],
       "solutionMetadata": {
-        "kind": "internal"
+        "kind": "autogen-label-from-site"
```
Comment (bqi343) [inline content/6_Advanced/SCC.problems.json:156]: site needs to be specified

File: content/6_Advanced/SCC.problems.json [hunk 2]
```diff
@@ -177,7 +179,7 @@
       "isStarred": false,
       "tags": ["2SAT", "DSU", "Sliding Window", "Greedy"],
       "solutionMetadata": {
-        "kind": "internal"
+        "kind": "none"
```
Comment (ryanchou-dev) [inline content/6_Advanced/SCC.problems.json:182]:
```suggestion
        "kind": "autogen-label-from-site",
        "site": "CC"
```
you can autogen for codechef as well

**Review by ryanchou-dev — APPROVED** [review]
i think we should set custom codechef text - afaik you have to access from discussion board?

File: content/6_Advanced/SCC.problems.json (hunk 2 above)
Comment (ryanchou-dev) [reply content/6_Advanced/SCC.problems.json:182]: actually never mind... i think only a couple of problems use it. i can still do the custom text for codechef though? wdyt

**Comment by bqi343** [comment]
```
error Problem cf-1089H claims to have an internal solution but doesn't
```

^ still this error

File: content/6_Advanced/SCC.problems.json (hunk 2 above)
Comment (SansPapyrus683) [reply content/6_Advanced/SCC.problems.json:182]: idk lol

**Pull request merged**